pyblish_deadline/plugins/select_deadline_write_nodes.py

```diff
@@ -61,4 +62,11 @@ def process(self, context):
                 plugin_data = plugin_data.copy()
                 plugin_data['WriteNode'] = node.name()
 
+                try:
+                    components = {node['fcompname'].getValue(): {}}
+                    instance.set_data('ftrackComponents', value=components)
```

Comment:
No, we are just submitting to Deadline and letting the event plugin handle that logic.